docs/getting-started-guides/docker-multinode.md

```diff
@@ -53,7 +49,9 @@ Please install Docker 1.6.2 or Docker 1.7.1.
 
 ## Prerequisites
 
-1. You need a machine with docker of right version installed.
+The only thing you need is a machine with **Docker 1.8.x or higher**
+
+It might work with older docker versions, but that isn't supported.
```

Comment:
@dalanlan Quite funny, I just edited it [on request](https://github.com/kubernetes/kubernetes/pull/18894#discussion_r48075919) by @brendandburns 

I think we should encourage users to use k8s with latest stable docker if possible.
Not everyone will use docker 1.8.x and higher, and as you say, 1.7.1 works well, so therefore I added that it _might work_ with older versions

But I don't know how Kubernetes Docker policy is set up, if docker 1.7.1 is supported by k8s 1.1.x